README.md

```diff
@@ -34,19 +34,35 @@ A summary of the currently supported features:
 
 Grab a Phabricator token at `https://<yourserver.com>/settings/panel/apitokens/`
 
-Configure:
+Configure in Linux:
 
      export PHAB_TOKEN=cli-ABC123
     --OR--
      echo "PHAB_TOKEN: cli-ABC123" > ~/.config/phabfive.yaml
 
+Configure in Windows:
+    Phabfive looks for configuration in the following sequence in Windows environment:
+      %ALLUSERSPROFILE%\phabfive\phabfive.yaml
+      %ALLUSERSPROFILE%\phabfive\phabfive.d\*.yaml
+      %LOCALAPPDATA%\phabfive\phabfive.yaml
+      %LOCALAPPDATA%\phabfive\phabfive.d\*.yaml
+
+    Make sure there is a minimum phabfive.yaml store in one of the location
+    e.g. echo "PHAB_TOKEN: cli-ABC123" > %LOCALAPPDATA%\phabfive\phabfive.yaml
```

Comment:
Is `echo` as a command generally available in a windows environment?

Reply:
Hi, I believe so: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/echo